webplatformcompat/jinja2/webplatformcompat/feature-basic.html

```diff
@@ -119,7 +119,7 @@
           {%- endif %}
         {%- else %}
 
-        ({{ support.support}})
+        (unknown)
```

Comment:
I'm a big fan of not localizing APIs. I've heard some compelling arguments, but generally I think it's a protocol and not a conversation and translating API strings leads to worse usability for the API. Maybe I haven't seen the right arguments, yet.

I don't have any experience with Ember.js, but @brittanystoroz said there was an ember-cli addon that made it really easy.